src/Doctrine/Orm/Extension/ParameterExtension.php

```diff
@@ -50,9 +50,11 @@ private function applyFilter(QueryBuilder $queryBuilder, QueryNameGeneratorInter
             }
 
             $filter = $this->filterLocator->has($filterId) ? $this->filterLocator->get($filterId) : null;
-            if ($filter instanceof FilterInterface) {
-                $filter->apply($queryBuilder, $queryNameGenerator, $resourceClass, $operation, ['filters' => $values, 'parameter' => $parameter] + $context);
+            if (!$filter instanceof FilterInterface) {
+                throw new \InvalidArgumentException(\sprintf('Could not find filter "%s" for parameter "%s" in operation "%s" for resource "%s".', $filterId, $parameter->getKey(), $operation?->getShortName(), $resourceClass));
```

Comment:
Can the exception be from `ApiPlatform\Metadata\Exception` please?

Reply:
Done ✅